packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -77,6 +101,20 @@ export interface ICXGenericResult {
   txid: string
 }
 
+/** ICX offer */
+export interface ICXOffer {
+  /** Transaction id of the order tx for which is the offer */
+  orderTx: string
+  /** Amount fulfilling the order */
+  amount: BigNumber
+  /** Address of DFI token and for receiving tokens in case of EXT/DFC order */
+  ownerAddress: string
+  // NOTE(surangap): c++ side this as number, but no type checks done. should be corrected from c++ side?
```

Comment:
you should open an issue on c++ side if you haven't already

Reply:
https://github.com/DeFiCh/ain/issues/567